rule.go

```diff
@@ -220,22 +229,22 @@ func escape(r string) string {
 }
 
 // within returns the within value for a specific content.
-func within(options []*ContentOption) int {
+func within(options []*ContentOption) string {
```

Comment:
why are we changing the return type here? the within value should always be an int, right?